packages/main/src/MultiComboBox.js

```diff
@@ -593,6 +600,39 @@ class MultiComboBox extends UI5Element {
 		this[`_handle${event.key}`] && this[`_handle${event.key}`](event);
 	}
 
+	_handleShow(event) {
+		const items = this.items;
+		const selectedItem = this._getSelectedItems()[0];
+		const focusedToken = this._tokenizer.tokens.find(token => token.focused);
+		const value = this.value;
+		const matchingItem = this.items.find(item => item.text.localeCompare(value, undefined, { sensitivity: "base" }) === 0);
+
+		if (this.readonly) {
+			return;
+		}
+
```

Comment:
Still when the component is in readonly mode, on Alt + Arrow Down the page is scrolled. You have to prevent the default behaviour even when the control is in readonly mode.